src/components/tickets/customToolbar/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { ReactComponent as PlusIcon } from '../../../assets/plus.svg';
+import TicketsModal from '../../modals/tickets';
+import { useSearchParams } from 'react-router-dom';
+import { Checkbox, FormControl, MenuItem } from '@mui/material';
+import {
+  FilterSelect,
+  PlusSpan,
+  SelectsContainer,
+  SortSelect,
+  StyledFilterAltIcon,
+  StyledFormControlLabel,
+  StyledGridToolbarContainer,
+  StyledInputLabel,
+  StyledSortIcon,
+  ViewButton,
+} from './styled';
+import { memo, useCallback, useState } from 'react';
+import { priorityOptions, sortingOptions } from './helper';
+
+export type IProps = {
+  apiUrl: string;
+  selectedPriorities: string[];
+  setSelectedPriorities: React.Dispatch<React.SetStateAction<string[]>>;
+};
+
+const CustomToolbar: React.FC<IProps> = ({ apiUrl, selectedPriorities, setSelectedPriorities }) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
put this comment `// eslint-disable-next-line @typescript-eslint/no-unused-vars` here using `// eslint-disable-line @typescript-eslint/no-unused-vars`

Reply:
fix:  at the place of  // eslint-disable-next-line @typescript-eslint/no-unused-vars   
 i placed 
/* eslint-disable @typescript-eslint/no-unused-vars */